Dockerfile

```diff
@@ -3,7 +3,7 @@ MAINTAINER Jason Wilder <jason@influxdb.com>
 
 ENV VERSION 0.4.0
 ENV DOWNLOAD_URL https://github.com/jwilder/docker-gen/releases/download/$VERSION/docker-gen-linux-amd64-$VERSION.tar.gz
-ENV DOCKER_HOST unix:///tmp/docker.sock
+ENV DOCKER_HOST unix:///var/run/docker.sock
```

Comment:
Yes it is.
The fix was at 6f32cc3f4c6c398028a6214abf4baf26a8a2aa52 which does not include that change.